bug: make mockery fails

This is occuring when I try to generate mocks for 


* #2150
* #1907 


Any ideas?  Maybe it's not a bug?



```log
make mockery
go generate -run="./scripts/mockery_generate.sh" ./...
31 Jan 24 23:57 IST INF couldn't read any config file version=v2.40.1
31 Jan 24 23:57 IST INF Starting mockery dry-run=false version=v2.40.1
31 Jan 24 23:57 IST INF Using config:  dry-run=false version=v2.40.1
31 Jan 24 23:57 IST WRN DEPRECATION: use of the packages config will be the only way to generate mocks in v3. Please migrate your config to use the packages feature. dry-run=false migration=https://vektra.github.io/mockery/v2.40/migrating_to_packages/ url=https://vektra.github.io/mockery/v2.40/features/#packages-configuration version=v2.40.1
31 Jan 24 23:57 IST INF Walking dry-run=false version=v2.40.1
31 Jan 24 23:57 IST INF Generating mock dry-run=false interface=Client qualified-name=github.com/cometbft/cometbft/abci/client version=v2.40.1
31 Jan 24 23:57 IST INF writing mock to file dry-run=false interface=Client qualified-name=github.com/cometbft/cometbft/abci/client version=v2.40.1
31 Jan 24 23:57 IST INF couldn't read any config file version=v2.40.1
31 Jan 24 23:57 IST INF Starting mockery dry-run=false version=v2.40.1
31 Jan 24 23:57 IST INF Using config:  dry-run=false version=v2.40.1
31 Jan 24 23:57 IST WRN DEPRECATION: use of the packages config will be the only way to generate mocks in v3. Please migrate your config to use the packages feature. dry-run=false migration=https://vektra.github.io/mockery/v2.40/migrating_to_packages/ url=https://vektra.github.io/mockery/v2.40/features/#packages-configuration version=v2.40.1
31 Jan 24 23:57 IST INF Walking dry-run=false version=v2.40.1
31 Jan 24 23:57 IST INF Generating mock dry-run=false interface=Application qualified-name=github.com/cometbft/cometbft/abci/types version=v2.40.1
31 Jan 24 23:57 IST INF writing mock to file dry-run=false interface=Application qualified-name=github.com/cometbft/cometbft/abci/types version=v2.40.1
31 Jan 24 23:57 IST INF couldn't read any config file version=v2.40.1
31 Jan 24 23:57 IST INF Starting mockery dry-run=false version=v2.40.1
31 Jan 24 23:57 IST INF Using config:  dry-run=false version=v2.40.1
31 Jan 24 23:57 IST WRN DEPRECATION: use of the packages config will be the only way to generate mocks in v3. Please migrate your config to use the packages feature. dry-run=false migration=https://vektra.github.io/mockery/v2.40/migrating_to_packages/ url=https://vektra.github.io/mockery/v2.40/features/#packages-configuration version=v2.40.1
31 Jan 24 23:57 IST INF Walking dry-run=false version=v2.40.1
31 Jan 24 23:57 IST INF Generating mock dry-run=false interface=BlockStore qualified-name=github.com/cometbft/cometbft/internal/evidence version=v2.40.1
31 Jan 24 23:57 IST INF writing mock to file dry-run=false interface=BlockStore qualified-name=github.com/cometbft/cometbft/internal/evidence version=v2.40.1
31 Jan 24 23:57 IST INF couldn't read any config file version=v2.40.1
31 Jan 24 23:57 IST INF Starting mockery dry-run=false version=v2.40.1
31 Jan 24 23:57 IST INF Using config:  dry-run=false version=v2.40.1
31 Jan 24 23:57 IST WRN DEPRECATION: use of the packages config will be the only way to generate mocks in v3. Please migrate your config to use the packages feature. dry-run=false migration=https://vektra.github.io/mockery/v2.40/migrating_to_packages/ url=https://vektra.github.io/mockery/v2.40/features/#packages-configuration version=v2.40.1
31 Jan 24 23:57 IST INF Walking dry-run=false version=v2.40.1
31 Jan 24 23:57 IST INF Generating mock dry-run=false interface=BlockStore qualified-name=github.com/cometbft/cometbft/internal/state version=v2.40.1
31 Jan 24 23:57 IST INF writing mock to file dry-run=false interface=BlockStore qualified-name=github.com/cometbft/cometbft/internal/state version=v2.40.1
31 Jan 24 23:57 IST INF couldn't read any config file version=v2.40.1
31 Jan 24 23:57 IST INF Starting mockery dry-run=false version=v2.40.1
31 Jan 24 23:57 IST INF Using config:  dry-run=false version=v2.40.1
31 Jan 24 23:57 IST WRN DEPRECATION: use of the packages config will be the only way to generate mocks in v3. Please migrate your config to use the packages feature. dry-run=false migration=https://vektra.github.io/mockery/v2.40/migrating_to_packages/ url=https://vektra.github.io/mockery/v2.40/features/#packages-configuration version=v2.40.1
31 Jan 24 23:57 IST INF Walking dry-run=false version=v2.40.1
31 Jan 24 23:57 IST INF Generating mock dry-run=false interface=EvidencePool qualified-name=github.com/cometbft/cometbft/internal/state version=v2.40.1
31 Jan 24 23:57 IST INF writing mock to file dry-run=false interface=EvidencePool qualified-name=github.com/cometbft/cometbft/internal/state version=v2.40.1
31 Jan 24 23:57 IST INF couldn't read any config file version=v2.40.1
31 Jan 24 23:57 IST INF Starting mockery dry-run=false version=v2.40.1
31 Jan 24 23:57 IST INF Using config:  dry-run=false version=v2.40.1
31 Jan 24 23:57 IST WRN DEPRECATION: use of the packages config will be the only way to generate mocks in v3. Please migrate your config to use the packages feature. dry-run=false migration=https://vektra.github.io/mockery/v2.40/migrating_to_packages/ url=https://vektra.github.io/mockery/v2.40/features/#packages-configuration version=v2.40.1
31 Jan 24 23:57 IST INF Walking dry-run=false version=v2.40.1
31 Jan 24 23:57 IST INF Generating mock dry-run=false interface=Store qualified-name=github.com/cometbft/cometbft/internal/state version=v2.40.1
31 Jan 24 23:57 IST INF writing mock to file dry-run=false interface=Store qualified-name=github.com/cometbft/cometbft/internal/state version=v2.40.1
31 Jan 24 23:57 IST INF couldn't read any config file version=v2.40.1
31 Jan 24 23:57 IST INF Starting mockery dry-run=false version=v2.40.1
31 Jan 24 23:57 IST INF Using config:  dry-run=false version=v2.40.1
31 Jan 24 23:57 IST WRN DEPRECATION: use of the packages config will be the only way to generate mocks in v3. Please migrate your config to use the packages feature. dry-run=false migration=https://vektra.github.io/mockery/v2.40/migrating_to_packages/ url=https://vektra.github.io/mockery/v2.40/features/#packages-configuration version=v2.40.1
31 Jan 24 23:57 IST INF Walking dry-run=false version=v2.40.1
31 Jan 24 23:57 IST INF Generating mock dry-run=false interface=BlockIndexer qualified-name=github.com/cometbft/cometbft/internal/state/indexer version=v2.40.1
31 Jan 24 23:57 IST INF writing mock to file dry-run=false interface=BlockIndexer qualified-name=github.com/cometbft/cometbft/internal/state/indexer version=v2.40.1
31 Jan 24 23:57 IST INF couldn't read any config file version=v2.40.1
31 Jan 24 23:57 IST INF Starting mockery dry-run=false version=v2.40.1
31 Jan 24 23:57 IST INF Using config:  dry-run=false version=v2.40.1
31 Jan 24 23:57 IST WRN DEPRECATION: use of the packages config will be the only way to generate mocks in v3. Please migrate your config to use the packages feature. dry-run=false migration=https://vektra.github.io/mockery/v2.40/migrating_to_packages/ url=https://vektra.github.io/mockery/v2.40/features/#packages-configuration version=v2.40.1
31 Jan 24 23:57 IST INF Walking dry-run=false version=v2.40.1
31 Jan 24 23:57 IST INF Generating mock dry-run=false interface=TxIndexer qualified-name=github.com/cometbft/cometbft/internal/state/txindex version=v2.40.1
31 Jan 24 23:57 IST INF writing mock to file dry-run=false interface=TxIndexer qualified-name=github.com/cometbft/cometbft/internal/state/txindex version=v2.40.1
31 Jan 24 23:57 IST INF couldn't read any config file version=v2.40.1
31 Jan 24 23:57 IST INF Starting mockery dry-run=false version=v2.40.1
31 Jan 24 23:57 IST INF Using config:  dry-run=false version=v2.40.1
31 Jan 24 23:57 IST WRN DEPRECATION: use of the packages config will be the only way to generate mocks in v3. Please migrate your config to use the packages feature. dry-run=false migration=https://vektra.github.io/mockery/v2.40/migrating_to_packages/ url=https://vektra.github.io/mockery/v2.40/features/#packages-configuration version=v2.40.1
31 Jan 24 23:57 IST INF Walking dry-run=false version=v2.40.1
31 Jan 24 23:57 IST INF Generating mock dry-run=false interface=StateProvider qualified-name=github.com/cometbft/cometbft/internal/statesync version=v2.40.1
31 Jan 24 23:57 IST INF writing mock to file dry-run=false interface=StateProvider qualified-name=github.com/cometbft/cometbft/internal/statesync version=v2.40.1
31 Jan 24 23:57 IST INF couldn't read any config file version=v2.40.1
31 Jan 24 23:57 IST INF Starting mockery dry-run=false version=v2.40.1
31 Jan 24 23:57 IST INF Using config:  dry-run=false version=v2.40.1
31 Jan 24 23:57 IST WRN DEPRECATION: use of the packages config will be the only way to generate mocks in v3. Please migrate your config to use the packages feature. dry-run=false migration=https://vektra.github.io/mockery/v2.40/migrating_to_packages/ url=https://vektra.github.io/mockery/v2.40/features/#packages-configuration version=v2.40.1
31 Jan 24 23:57 IST INF Walking dry-run=false version=v2.40.1
31 Jan 24 23:57 IST INF Generating mock dry-run=false interface=LightClient qualified-name=github.com/cometbft/cometbft/light/rpc version=v2.40.1
31 Jan 24 23:57 IST INF writing mock to file dry-run=false interface=LightClient qualified-name=github.com/cometbft/cometbft/light/rpc version=v2.40.1
31 Jan 24 23:57 IST INF couldn't read any config file version=v2.40.1
31 Jan 24 23:57 IST INF Starting mockery dry-run=false version=v2.40.1
31 Jan 24 23:57 IST INF Using config:  dry-run=false version=v2.40.1
31 Jan 24 23:57 IST WRN DEPRECATION: use of the packages config will be the only way to generate mocks in v3. Please migrate your config to use the packages feature. dry-run=false migration=https://vektra.github.io/mockery/v2.40/migrating_to_packages/ url=https://vektra.github.io/mockery/v2.40/features/#packages-configuration version=v2.40.1
31 Jan 24 23:57 IST INF Walking dry-run=false version=v2.40.1
31 Jan 24 23:57 IST INF Generating mock dry-run=false interface=Mempool qualified-name=github.com/cometbft/cometbft/mempool version=v2.40.1
31 Jan 24 23:57 IST INF writing mock to file dry-run=false interface=Mempool qualified-name=github.com/cometbft/cometbft/mempool version=v2.40.1
31 Jan 24 23:57 IST INF couldn't read any config file version=v2.40.1
31 Jan 24 23:57 IST INF Starting mockery dry-run=false version=v2.40.1
31 Jan 24 23:57 IST INF Using config:  dry-run=false version=v2.40.1
31 Jan 24 23:57 IST WRN DEPRECATION: use of the packages config will be the only way to generate mocks in v3. Please migrate your config to use the packages feature. dry-run=false migration=https://vektra.github.io/mockery/v2.40/migrating_to_packages/ url=https://vektra.github.io/mockery/v2.40/features/#packages-configuration version=v2.40.1
31 Jan 24 23:57 IST INF Walking dry-run=false version=v2.40.1
31 Jan 24 23:57 IST INF Generating mock dry-run=false interface=Peer qualified-name=github.com/cometbft/cometbft/p2p version=v2.40.1
31 Jan 24 23:57 IST INF writing mock to file dry-run=false interface=Peer qualified-name=github.com/cometbft/cometbft/p2p version=v2.40.1
31 Jan 24 23:57 IST INF couldn't read any config file version=v2.40.1
31 Jan 24 23:57 IST INF Starting mockery dry-run=false version=v2.40.1
31 Jan 24 23:57 IST INF Using config:  dry-run=false version=v2.40.1
31 Jan 24 23:57 IST WRN DEPRECATION: use of the packages config will be the only way to generate mocks in v3. Please migrate your config to use the packages feature. dry-run=false migration=https://vektra.github.io/mockery/v2.40/migrating_to_packages/ url=https://vektra.github.io/mockery/v2.40/features/#packages-configuration version=v2.40.1
31 Jan 24 23:57 IST INF Walking dry-run=false version=v2.40.1
31 Jan 24 23:57 IST INF Generating mock dry-run=false interface=AppConnConsensus qualified-name=github.com/cometbft/cometbft/proxy version=v2.40.1
31 Jan 24 23:57 IST INF writing mock to file dry-run=false interface=AppConnConsensus qualified-name=github.com/cometbft/cometbft/proxy version=v2.40.1
31 Jan 24 23:57 IST INF Generating mock dry-run=false interface=AppConnMempool qualified-name=github.com/cometbft/cometbft/proxy version=v2.40.1
31 Jan 24 23:57 IST INF writing mock to file dry-run=false interface=AppConnMempool qualified-name=github.com/cometbft/cometbft/proxy version=v2.40.1
31 Jan 24 23:57 IST INF Generating mock dry-run=false interface=AppConnQuery qualified-name=github.com/cometbft/cometbft/proxy version=v2.40.1
31 Jan 24 23:57 IST INF writing mock to file dry-run=false interface=AppConnQuery qualified-name=github.com/cometbft/cometbft/proxy version=v2.40.1
31 Jan 24 23:57 IST INF Generating mock dry-run=false interface=AppConnSnapshot qualified-name=github.com/cometbft/cometbft/proxy version=v2.40.1
31 Jan 24 23:57 IST INF writing mock to file dry-run=false interface=AppConnSnapshot qualified-name=github.com/cometbft/cometbft/proxy version=v2.40.1
31 Jan 24 23:57 IST INF couldn't read any config file version=v2.40.1
31 Jan 24 23:57 IST INF Starting mockery dry-run=false version=v2.40.1
31 Jan 24 23:57 IST INF Using config:  dry-run=false version=v2.40.1
31 Jan 24 23:57 IST WRN DEPRECATION: use of the packages config will be the only way to generate mocks in v3. Please migrate your config to use the packages feature. dry-run=false migration=https://vektra.github.io/mockery/v2.40/migrating_to_packages/ url=https://vektra.github.io/mockery/v2.40/features/#packages-configuration version=v2.40.1
31 Jan 24 23:57 IST INF Walking dry-run=false version=v2.40.1
31 Jan 24 23:57 IST INF Generating mock dry-run=false interface=ClientCreator qualified-name=github.com/cometbft/cometbft/proxy version=v2.40.1
31 Jan 24 23:57 IST INF writing mock to file dry-run=false interface=ClientCreator qualified-name=github.com/cometbft/cometbft/proxy version=v2.40.1
.git/logs/refs/remotes/faddat/faddat/refactor-node_test.go:1:1: expected 'package', found 0000000000000000000000000000000000000000
.git/logs/refs/remotes/faddat/faddat/refactor-node_test.go:1:44: exponent has no digits
.git/logs/refs/remotes/faddat/faddat/refactor-node_test.go:1:112: illegal character U+0040 '@'
.git/refs/remotes/faddat/faddat/refactor-node_test.go:1:3: exponent has no digits
rpc/jsonrpc/client/integration_test.go
exit status 2
make: *** [mockery] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -112,7 +112,6 @@ testdata-metrics:
 #? mockery: Generate test mocks
 mockery:
 	go generate -run="./scripts/mockery_generate.sh" ./...
-	@go run mvdan.cc/gofumpt@latest -l -w .
 .PHONY: mockery
 
 ###############################################################################

